hastexo/hastexo.py

```diff
@@ -91,6 +91,12 @@ class HastexoXBlock(XBlock,
         scope=Scope.settings,
         help="The relative path to an uploaded executable script. "
              "For example, \"hot_script.sh\".")
+    delete_age = Integer(
```

Comment:
So, down the road we want to support not only a plain number that we interpret as seconds, but also suffixes like `3d` for three days and `1w` for one week.

Thus, I am guessing that you want this to be a String, which for now you then internally *cast* to an integer. Does that make sense?

Reply:
Yes, makes sense.